Modules/xASL_module_Population.m

```diff
@@ -48,7 +48,17 @@
     return;
 end
 
-x = xASL_wrp_Population_PrepareAtlas4ROI(x); % Parse x.S.Atlases & x.S.TissueMasking
+% Check again for Atlases (main checking is done when loading dataPar)

```

Comment:
I still tricky because "advanced" users my just run without specifying in the dataPar, start debugging mode here, and provide the x.S.Atlases & x.S.TissueMasking here. 

Perhaps it is best practice to move the checking we did here to a separate function, and calling this upon dataPar loading as well as just before running xASL_wrp_GetRoi...?

Reply:
If the user doesn't provide dataPar, then default values will be used. So if the same user puts a breakpoint here and adds incorrect values here, it will check it below and create an error (so exit correctly), but it won't print him the manual on how to do this correctly.

I'd say that if somebody does anything like that, he doesn't need to have the manual printed. I don't think the full check function is needed here again as we have the full check of input validity here, which should be enough....

Reply:
Also fine, I think in general it doesn't hurt to have such checks twice (hardly any time inefficiency but more warnings can still help us avoid explaining stuff). But you can choose and resolve.